native-image-workshop/README.md

```diff
@@ -237,20 +254,13 @@ This generates an error:
 Exception in thread "main" java.lang.NoClassDefFoundError
         at org.apache.log4j.Category.class$(Category.java:118)
         at org.apache.log4j.Category.<clinit>(Category.java:118)
-        at com.oracle.svm.core.hub.ClassInitializationInfo.invokeClassInitializer(ClassInitializationInfo.java:350)
-        at com.oracle.svm.core.hub.ClassInitializationInfo.initialize(ClassInitializationInfo.java:270)
-        at java.lang.Class.ensureInitialized(DynamicHub.java:496)
-        at com.oracle.svm.core.hub.ClassInitializationInfo.initialize(ClassInitializationInfo.java:235)
-        at java.lang.Class.ensureInitialized(DynamicHub.java:496)
+        at java.lang.Class.ensureInitialized(DynamicHub.java:552)
         at oracle.ListDir.<clinit>(ListDir.java:75)
-        at com.oracle.svm.core.hub.ClassInitializationInfo.invokeClassInitializer(ClassInitializationInfo.java:350)
-        at com.oracle.svm.core.hub.ClassInitializationInfo.initialize(ClassInitializationInfo.java:270)
-        at java.lang.Class.ensureInitialized(DynamicHub.java:496)
         at oracle.App.main(App.java:63)
 Caused by: java.lang.ClassNotFoundException: org.apache.log4j.Category
-        at com.oracle.svm.core.hub.ClassForNameSupport.forName(ClassForNameSupport.java:60)
-        at java.lang.Class.forName(DynamicHub.java:1211)
-        ... 12 more
+        at java.lang.Class.forName(DynamicHub.java:1433)
+        at java.lang.Class.forName(DynamicHub.java:1408)
+        ... 5 more
 ```
 
 Why? This is caused by the addition of the Log4J library. This library depends heavily upon the reflection calls,
```

Comment:
Why? This is caused by the addition of the Log4J library.  > This is caused by the addition of the Log4J library.

Reply:
I like what I had :) Would prefer to keep it. It's more personal

Reply:
Sure, just a suggestion :)